### PR TITLE
fix: unit tests and reenable running

### PR DESCRIPTION
**Pull request by seve**
#### Reviewers
**Functional:**  @tihuan 

---

A bump in our jest version began running our unit tests in a node environment rather than a browser.  This PR adds SSR checks in various places in order to not make attempts to call window where there is none present.  Also re-enables unit tests on the push_tests workflow.

### CODE REVIEW

File: client/src/components/framework/toasters.js [hunk 1]
```diff
@@ -1,31 +1,33 @@
 import { Position, Toaster, Intent } from "@blueprintjs/core";
 
 /** Singleton toaster instance. Create separate instances for different options. */
-
-const ToastTopCenter = Toaster.create({
-  className: "recipe-toaster",
-  position: Position.TOP,
-  maxToasts: 4,
-});
+let ToastTopCenter = null;
```
Comment (tihuan) [inline client/src/components/framework/toasters.js:4]: is this still needed now we have JSDom? Thanks!

File: client/src/components/framework/toasters.js [hunk 2]
```diff
@@ -1,31 +1,33 @@
...
+if (typeof document !== "undefined") {
```
Comment (tihuan) [inline client/src/components/framework/toasters.js:5]: same here!

File: client/__tests__/util/annoMatrix/serverMocks/index.js [hunk 3]
```diff
@@ -1,11 +1,4 @@
 export const baseDataURL = "https://a.fake.url/api/v0.2";
 
-window.CELLXGENE = {
```
Comment (tihuan) [inline client/__tests__/util/annoMatrix/serverMocks/index.js:3]: likely can be reverted too?

File: client/src/globals.js [hunk 4]
```diff
@@ -102,7 +102,7 @@ const CXG_SERVER_PORT =
 
 let _API;
 
-if (window.CELLXGENE && window.CELLXGENE.API) {
+if (typeof window !== "undefined" && window.CELLXGENE && window.CELLXGENE.API) {
```
Comment (tihuan) [inline client/src/globals.js:105]: same here!

File: client/src/components/framework/toasters.js (hunk 1 above)
Comment (seve) [reply client/src/components/framework/toasters.js:4]: I dropped JSDom on the unit tests since the tests themselves don't need any browser API's

**Review by tihuan — APPROVED** [review]
Ahh I see what you mean now 👍 That makes sense! Thank you!

**Pull request merged**